opm/material/common/Tabulated1DFunction.hpp

```diff
@@ -268,7 +268,7 @@ class Tabulated1DFunction
         Scalar y0 = yValues_[segIdx];
         Scalar y1 = yValues_[segIdx + 1];
 
-        return y0 + (y1 - y0)*(x - x0)/(x1 - x0);
+        return y0 + (y1 - y0)/(x1 - x0)*(x - x0);
```

Comment:
This change causes changes to numerical results, and I do not see any reason for the change. I'd therefore like you to revert it.